Makefile

```diff
@@ -893,7 +900,7 @@ sha256_bool: install_rs_check_toolchain
 pcc: no_tfhe_typo no_dbg_log check_fmt lint_doc clippy_all check_compile_tests
 
 .PHONY: pcc_gpu # pcc stands for pre commit checks for GPU compilation
-pcc_gpu: clippy_gpu clippy_cuda_backend check_compile_tests_gpu
+pcc_gpu: clippy_gpu clippy_cuda_backend check_compile_tests_gpu build_cuda_backend
```

Comment:
Instead of `build_cuda_backend`, I think it would be more consistent with what @IceTDrinker did to have `check_compile_build_gpu`, just like the `check_compile_tests_gpu`, or to have just a `check_compile_tests_benches_gpu` maybe.